### ISSUE TITLE
Resize() not working for width on customDialog

### ISSUE THREAD

**Issue by bpasham** [microsoft/azure-devops-extension-sdk#18]
Resize() not working for width on customDialog. Height of the dialog is changing however there doesn't seem to have a way to change the width of the dialog.

**Comment by kp101090** [comment]
It has been 6 months. Anyone looking into this? Can someone acknowledge at the least that this is an issue?
customDialog aize seems to be dictated by "bolt-callout-medium" and this width of the customDialog does not change on SDK.resize()

**Comment by karelkral** [comment]
Yes, I have the same issue. Sadly, development for Azure DevOps is very frustrating. No support from MS!

**Comment by liminany** [comment]
I have same probelm, hope fix .

**Comment by AshwaniT** [comment]
I am facing similar issue.  I don't know why we always apply  .bolt-callout-medium class to outer div . this class has width 480px so even if you try using sdk.resize it doesn't work.  no way to override this class width

**Comment by matt-psaltis** [comment]
Adding the yearly me too comment. Pretty please DevOps team!?

**Comment by TechnoRahmon** [comment]
hey same issue with me, any update ?

**Comment by pbcahill** [comment]
I am also dealing with the same issue.

**Comment by ug-008** [comment]
These has been an issue for 3years now and it's depriving me of migration to the new SDK

**Comment by ug-008** [comment]
Team, please fix this...

**Comment by kantenkugel** [comment]
Still seems to be an issue btw.
The class .bolt-callout-medium on one of the parents (of the iframe) is restricting the width to 480px, so the inner content (from eg a contribution) can never be > approx 440px, which makes writing forms that fit that an unnecesarily stupid hassle

**Comment by kovacevicemir** [comment]
Any news please?

**Comment by eidermauricio** [comment]
news?

**Comment by Cyb3rP4w** [comment]
I have the same issue :/ any news?
